src/views/common/Animation/AnimationControls/PlaybackOptionalButtons.ts

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
This file collects the set of buttons which can optionally be included (and whose presence/absence will dynamically change the width of the right-hand control panel).